backend/company_api.py

```
@@ -936,7 +936,154 @@ async def get_public_doctor_report():
-# EXPORT ROUTER
+# SKILLS ENDPOINTS
 # =============================================================================
-# This will be included in backend/server.py
-# app.include_router(router)
+
+@router.get("/skills")
+async def list_skills(
+    family: str | None = Query(None, description="Filter by specialist family"),
+    category: str | None = Query(None, description="Filter by skill category"),
+    search: str | None = Query(None, description="Search by name/description"),
+    user: dict = Depends(_get_current_user_thunk),
```

Ruff (0.15.15)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)